Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -22,72 +23,68 @@ BatteryStateOfChargeService::~BatteryStateOfChargeService()
 
 double BatteryStateOfChargeService::totalAmpHoursUsed() const
 {
-
     return totalAmphoursUsed_;
 }
 
 bool BatteryStateOfChargeService::isCharging() const
 {
-    if(current_ < 0){
-        return true;
-    }
-    return false;
+    return (current_ < 0);
 }
 
 QTime BatteryStateOfChargeService::timeWhenChargedOrDepleted() const
 {
-    double totalHoursRemaining, msRemaining;
-        int h, m, s, ms;
+        double totalHoursRemaining;
+        double millisecRemaining;
+        int hour;
+        int min;
+        int sec;
+        int millisec;
```

Comment:
These aren't used until later, and can be set right away:
`int hour =  totalHoursRemaining;`, etc.